openstf/tasks/create_forecast.py

```diff
@@ -36,7 +36,7 @@
 from openstf.tasks.utils.taskcontext import TaskContext
 
 T_BEHIND_DAYS: int = 14
-T_AHEAD_DAYS: int = 3
+T_AHEAD_DAYS: int = 2
```

Comment:
Not wrong necessarily, but why is this variable changed in this PR?

Reply:
Before we specified the forecast horizon in 48 hours from datetime.now() when calling generate_forecast_datetime_range(). 

Now we have a dynamic forecast range based on the forecast data, and we don't use datetime.now()  and forecast horizon of 48 hours anymore. If we want to keep a forecast horizon of 48 hours, we should limit T_AHEAD_DAYS to 2 days so we retrieve data that is 2 days in the future. If we don't change this value, we will retrieve forecast data 3 days into the future and all forecasts will be 3 days ahead instead of 2.

Reply:
@FrankKr Can you resolve this conversation if it's clear? Or if changes are still required, it would be nice to hear!

Reply:
Ok, makes sense!